core/deployment/src/main/java/io/quarkus/deployment/builditem/InitalizationCompletedBuildItem.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class InitalizationCompletedBuildItem extends MultiBuildItem {
```

Comment:
We definitely need some Javadoc on this. Also, I am wondering if the name is too generic... Maybe we need to be specific about what kind of initialization we are talking about.

Reply:
How about `InitializationTaskCompleteBuildItem`?